Packages/com.unity.inputsystem.tests/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [0.2.6-preview] - 2019-04-14
+
+Test split off into separate package
```

Comment:
Do we need a separate CHANGELOG for this one? I assume it's mainly a technicality required by package verification but what if we just add a link to the main CHANGELOG and leave it at that?

Reply:
Yes, I had to add it due to the verification. But, good point, if we can do this using a symlink, then that would make it easier to maintain. Will give it a try.